.github/workflows/binary-compatibility.yml

```diff
@@ -23,16 +29,16 @@ jobs:
           path: ${{ github.run_id }}.${{ github.base_ref }}
       - name: Build `${{ github.base_ref }}`
         run: >
-          ./mvnw -B -V --no-transfer-progress -e -f ${{ github.run_id }}.${{ github.base_ref }}/pom.xml package 
+          ./mvnw -B -V --no-transfer-progress -e -f ${{ github.run_id }}.${{ github.base_ref }}/assertj-core/pom.xml package 
           -DskipTests
           -Djansi.passthrough=true
           -Dstyle.color=always
       - name: Compare `${{ github.head_ref }}` with `${{ github.base_ref }}`
         run: >
-          ./mvnw -B -V --no-transfer-progress -e -Pjapicmp-branch package japicmp:cmp
+          ./mvnw -B -V --no-transfer-progress -e -f assertj-core/pom.xml -Pjapicmp-branch package japicmp:cmp
           -DskipTests
           -Djapicmp.breakBuildOnBinaryIncompatibleModifications=true
-          -Djapicmp.oldVersion.basedir=${{ github.run_id }}.${{ github.base_ref }}
+          -Djapicmp.oldVersion.basedir=${{ github.run_id }}.${{ github.base_ref }}/assertj-core
           -Djansi.passthrough=true
           -Dstyle.color=always
```

Comment:
The binary compatibility check with the `main` branch fails because it tries to look already at the new path.

Reply:
ok